packages/react-components/src/components/gauge/gaugeDataQualityText.tsx

```diff
@@ -6,20 +6,15 @@ import { DataQualityText } from '../data-quality/data-quality-text';
 export const GaugeDataQualityText = ({
   error,
   quality,
-  showName,
 }: {
   error?: string;
   quality?: Quality;
-  showName?: boolean;
 }) => {
   if (error || !quality) {
     return null;
   }
   return (
-    <div
-      className='gauge-data-quality'
-      style={{ bottom: showName ? '36%' : '40%' }}
-    >
+    <div className='gauge-data-quality' style={{ bottom: '2%' }}>
```

Comment:
since we are using percentages, can you add some screenshots of this widget on the dashboard at various sizes. small / medium / large square, and then similarly, small / medium / large rectangles where we test short but wide rectangles / tall but skinny rectangles.